Eastern-Research-Group/mywaterway
# Feature/streamgage updates

## Related Issues:
* https://jira.epa.gov/browse/HMW-4

## Main Changes:
A few updates to the USGS streamgages, to handle some edge cases with the returned data, and how its displayed:
* Adds USGS name alongside the USGS parameter code for each parameter/category in the streamgage measurements table.
* Displays a "no data available" message for streamgages who don't return any measurements.
* Only display the show more/less button (that's rendered before the secondary measurements) if there's at least one primary measurement; otherwise, the secondary measurements are just shown
* Updates the show more/less button's styles to better align visually with the existing show more/less buttons and the existing accordions.

## Steps To Test:
1. Navigate to http://localhost:3000/community/051100040306/overview
2. Switch to the "Monitoring & Sensors" subtab, and toggle on the "Current Water Conditions" switch
3. Expand the first streamgage ("ROUGH RIVER DAM LAKESIDE STATION 1A") and verify the USGS parameter name is displayed after the parameter code (below the HMW name). Also notice the updated styles of the "Show more/less categories" button
4. Expand the next streamgage ("Rough River Dam RRD OW14A RRD"). This one has no "primary" parameters/categories (only secondary), so verify that there's no "Show more/less categories" button – only the list of secondary parameters.
5. Navigate to http://localhost:3000/community/010900010702/overview and expand the last streamgage ("Upper Hobbs Brook Reservoir Buoy 1"). This one doesn't return any measurement data, so ensure "No data available." is displayed in place of the latest measurements table.
